goal_src/jak2/levels/fortress/ammo_dump/fort-robotank-turret.gc

```diff
@@ -258,8 +258,9 @@
           (s5-0 (new 'stack-no-clear 'vector))
           (gp-0 (new 'stack-no-clear 'vector))
           (f30-0 4096.0)
+          (ratio (* (-> *pc-settings* aspect-ratio) 0.375))
           )
-      (let ((f0-4 (* 0.00013563369 (tan (* 0.5 (-> *math-camera* fov))) f30-0)))
+      (let ((f0-4 (* 0.00013563369 (tan (* ratio (-> *math-camera* fov))) f30-0)))
```

Comment:
tag as changed for PC.

Also, shouldn't you do `(* (-> *pc-settings* aspect-ratio-scale) 0.5)` here?